includes/abstracts/class-wpum-wp-db-table.php

```diff
@@ -0,0 +1,357 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected vipgoci-phpcs-scan-tlwvud.php, but found vipgoci-phpcs-scan-TLWVud.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).